src/99-appendices/09-qmri.md

```diff
@@ -0,0 +1,477 @@
+
+In the future, novel qMRI applications will be introduced that are not yet
+described by the BIDS. If such applications can not be interpreted as a 
+subset of a pre-existing `grouping suffix`,
+a new grouping suffix can be introduced, but should adhere to the following
+principles:
+
+* All grouping suffixes MUST be capitalized.
+* Grouping suffixes MUST attain a clear description of the qMRI application
+that they relate to. Hyperlinks to example applications and/or more detailed
+descriptions are encouraged whenever possible.
+* Unless the pulse sequence is exclusively associated with a specific qMRI
+application (e.g. `MP2RAGE`), sequence names are NOT used as grouping suffixes.
+* If it is possible to derive a qMRI application from an already existing
+grouping suffix by defining a set of logical conditions over the metadata
+fields, the [table of method-specific priority levels](#prioritylevels) and the
+[table of qMRI applications that can be derived from an existing grouping suffix](#varianttable)
+MUST be expanded instead of introducing a new grouping suffix.
+Please visit the [JSON content for grouping suffixes](#whichmetadata) 
+for further details.
+* Please note that if a structural data has the type of grouped scan collection,
+the use of `_suffix` alone cannot distinguish its members from each other,
+failing to identify their roles as inputs to the calculation of qMRI maps.
+Although such images are REQUIRED to be grouped by a proper grouping suffix,
+they are also RECOMMENDED to include at least one of the `acq`, `part`, `echo`,
+`met`, `inv`-key/value-pairs  to indicate which scanning parameters are 
+systematically altered.
+
+
+# Management of the qMRI maps
+
+All qMRI maps are generated following a set of calculations. Unlike conventional 
+MR images, they are not products of an MRI image reconstruction (from k-space data 
+to structural images). There are two possible options in the way a qMRI map is obtained: 
+
+1. Pre-generated qMRI maps: The qMRI maps are generated immediately after the 
+reconstruction of the multi-contrast input images and made available to the user
+at the scanner console. The acquisition scenarios may include (a) vendor pipelines 
+or (b) open-source pipelines deployed at the scanner site.
+2. Post-generated qMRI maps: The qMRI maps are generated from the multi-contrast
+input images after they are exported outside of the scanner site.
+
+## Where to place qMRI maps? 
+
+**If the provenance record of the qMRI map generation is NOT accessible:**
+
+Although qMRI maps are derivatives by definition, we cannot relate them to their 
+multi-contrast input images in this case. Therefore, qMRI maps lacking provenance
+are directly placed at the `/sub-#/anat` directory.
+
+**If the provenance record of the qMRI map generation is available:**
+
+In this case, qMRI maps SHOULD be stored in the `derivatives` folder, but MAY
+be symbolic linked to the corresponding raw data directory to facilitate the
```

Comment:
There are pretty strong arguments against symbolic linking (please see #526). One solution that @effigies proposes is to use the dataset description file to characterize the dataset as derivatives. I don't think there's any room for a distinction based on provenance in this case.

Reply:
Thank you @tsalo for the pointer, I was following that discussion and planning to bring this up in our next BEP001 meeting. We will soon have another meeting before taking actions to address comments and implement changes I suggested above.